dashboard/test/ui/features/plc/courseUnitNavigation.feature

```diff
@@ -0,0 +1,30 @@
+@dashboard_db_access
+Feature: Basic navigation for PLC stuff
+
+Background:
+  Given I am a teacher
+  And I am enrolled in a plc course
+  Given I am on "http://studio.code.org/course/csp-support"
+  And I wait to see ".course_unit_sections"
+
+
+Scenario: Basic navigation and ribbon changing works as expected
+  Then elements ".ribbon" have css property "background-color" equal to "rgb(255, 255, 255), rgb(255, 255, 255), rgb(255, 255, 255)"
+  Then I click selector ".course_unit_title a"
+  And I get redirected to "/s/teachercon" via "dashboard"
+  Then I fake completion of the assessment
+  Then I click selector "#breadcrumb a"
+  And I get redirected to "/course/csp-support" via "dashboard"
+  And I wait to see ".course_unit_sections"
+  Then element ".course_unit_section a:nth-child(3) .ribbon" has css property "background-color" equal to "rgb(14, 190, 14)"
```

Comment:
I suspect you're right, without getting into something nasty like parsing that file. How about just defining local constants so at least they have readable names, and add a comment saying colors.scss is the source of truth (in case it breaks in the future).